lib/importers/licence_transaction/licence_importer.rb

```diff
@@ -0,0 +1,83 @@
+module Importers
+  module LicenceTransaction
+    class LicenceImporter
+      def call
+        filtered_licences.each do |licence|
+          details = licence["details"]
+          new_licence = ::LicenceTransaction.new(
+            locale: "en",
+            base_path: "/find-licences#{licence['base_path']}",
+            title: licence["title"],
+            summary: licence["description"],
+            body: details["licence_overview"].first["content"],
+            publication_state: "draft",
+            state_history: [],
+            last_edited_at: licence["last_edited_at"],
+            public_updated_at: licence["public_updated_at"],
+            first_published_at: licence["first_published_at"],
+            update_type: "major",
+            licence_transaction_location: [],
+            licence_transaction_industry: [],
+            licence_transaction_will_continue_on: details["will_continue_on"],
+            licence_transaction_continuation_link: details["continuation_link"],
+            licence_transaction_licence_identifier: details["licence_identifier"],
+            imported: true,
+          )
+
+          new_licence.change_note = "Imported from Publisher"
+
+          unless new_licence.valid?
+            puts "[ERROR] licence: #{new_licence.base_path} has validation errors: #{new_licence.errors.inspect}"
+            next
+          end
+
+          new_content_id = new_licence.content_id
+
+          save_draft(new_licence)
+
+          save_links(licence["content_id"], new_content_id)
+
+          publish(new_content_id)
+
+          puts "Published: #{new_licence.base_path}"
```

Comment:
Not a blocker, but I'm surprised the linter hasn't picked this up. It's definitely something that Rubocop warns about: https://www.rubydoc.info/gems/rubocop-rails/RuboCop/Cop/Rails/Output

Reply:
I did have a little play but couldn't get the correct logging behaviour so leaving as is. This is only rule I find quite annoying / adds little value in these sort of scenarios. We do use `puts` in various other places in the app too.

Reply:
I got caught by the linter in the licence-finder PR and ended up having to use `Rails.logger.info`. In development it output to the log file. But yes, it's very irritating.